workflow/rules/common.smk

```diff
@@ -1407,11 +1407,11 @@ def get_reads_by_stage(wildcards):
     if wildcards.stage == "raw":
         return get_fastqs(wildcards)
     elif wildcards.stage == "trimmed":
-        return "results/{date}/trimmed/porechop/adapter_barcode_trimming/{sample}.fastq"
+        return "results/{date}/norm_trim_corr_reads/{sample}/{sample}.correctedReads.clip.fasta"
```

Comment:
wouldn't it be more common to stay with fastq files here?

Reply:
I don't see why, since qf is done before and qual information not used again from hereon afaik. So would just be bigger files.